cl/custom_filters/templatetags/text_filters.py

```diff
@@ -121,14 +121,12 @@ def naturalduration(seconds, autoescape=None, as_dict=False):
         61 --> 1:01
         3602 --> 1:00:02
     """
-    if seconds is None:
+    try:
+        seconds = int(seconds)
+    except ValueError:
+        seconds = 0
+    except TypeError:
```

Comment:
What's the story here?

Reply:
.... Probably a bad decision.  Updated and reverted.  minus some part that was not used.